bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -586,21 +587,32 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       new Problem(Severity.Todo, "Please create observations with conditions, targets, and resources.", "Observations", ())
     }
 
+    private def investigatorFullName(i: Investigator, default: String = "Investigator"): String = {
+      val piEmptyName = PrincipalInvestigator.empty.fullName
+      i.fullName.trim match {
+        case ""            => default
+        case `piEmptyName` => "PI"
```

Comment:
I think that instead of the quotes you can make the variable be capitalized and get the same effect

``
...
   case PiEmptyName => ...
...
```